### ISSUE TITLE
Install doesn't work for PackageRefernces with dependencies (package spec problem)

### ISSUE THREAD

**Issue by drewgillies** [NuGet/Home#4015]
Repro Steps:
1.	New a C# Console App (.Net Framwork).
2.	Add the following properties to the property group at the top of the csproj file:
`<RuntimeIdentifiers Condition="'$(RuntimeIdentifiers)'==''">win</RuntimeIdentifiers>`
3.	Add the following reference to the csproj file, before the final targets import line starting with 
```
<Import Project="$(MSBuild...
<ItemGroup> 
        <PackageReference Include="Newtonsoft.Json"> 
<Version>9.0.1</Version> 
        </PackageReference> 
</ItemGroup>
```
From Rob: (by the way, version as attribute on the packagereference is the preferred form...please ensure that the csproj project system defaults to that serialization form when adding packages. ideally, it also preserves whichever form the user had in the csproj)

4.	Reload the project in VS and run solution restore.
5.	Open PM UI and install package ‘NUnit’ 3.5.0
6.	Observe the ‘Installed’ tab.

Expected:
Installed packages show in installed tab.

Actual:
Nothing show in installed tab.

**Comment by drewgillies** [comment]
Raised by Andy T in legacy csproj scenario testing. I'm not convinced this is a regression as I've seen the blank Install tab in UWP/console on occasion before (but haven't deduced repro steps). But this is a solid repro.

**Comment by jainaashish** [comment]
Merged to Dev #17fe755c4d8bc3f50fe1de53b26bae0976db10a7 Waiting for RC2 decision

**Comment by rrelyea** [comment]
Let's do this in RC3. So go ahead and close.
